src/lib/installation/widgets/lsm.rb

```diff
@@ -0,0 +1,110 @@
+require "yast"
+require "cwm/custom_widget"
+require "cwm/replace_point"
+require "cwm/common_widgets"
+require "installation/widgets/selinux_mode"
+
+Yast.import "HTML"
+
+module Installation
+  module Widgets
+    class LSM < CWM::CustomWidget
```

Comment:
NP: let's document the widget a bit. Something like

```suggestion
    # Widget for managing the LSM configuration during installation
    class LSM < CWM::CustomWidget
```

or better.